go/memcached-operator/go.mod

```diff
@@ -3,36 +3,37 @@ module github.com/operator-framework/operator-sdk-samples/go/memcached-operator
 go 1.13
 
 require (
```

Comment:
Error to build the image: 

```
 $ operator-sdk build $IMAGE
build github.com/operator-framework/operator-sdk-samples/go/memcached-operator/cmd/manager: cannot load k8s.io/api/apps: module k8s.io/api@latest found (v0.17.2, replaced by k8s.io/api@v0.0.0-20191016110408-35e52d86657a), but does not contain package k8s.io/api/apps
Error: failed to build operator binary: (failed to exec []string{"go", "build", "-o", "/Users/camilamacedo/go/src/github.com/operator-framework/operator-sdk-samples/go/memcached-operator/build/_output/bin/memcached-operator", "-gcflags", "all=-trimpath=/Users/camilamacedo/go/src/github.com/operator-framework/operator-sdk-samples/go", "-asmflags", "all=-trimpath=/Users/camilamacedo/go/src/github.com/operator-framework/operator-sdk-samples/go", "github.com/operator-framework/operator-sdk-samples/go/memcached-operator/cmd/manager"}: exit status 1)
Usage:
  operator-sdk build <image> [flags]

Flags:
      --go-build-args string      Extra Go build arguments as one string such as "-ldflags -X=main.xyz=abc"
  -h, --help                      help for build
      --image-build-args string   Extra image build arguments as one string such as "--build-arg https_proxy=$https_proxy"
      --image-builder string      Tool to build OCI images. One of: [docker, podman, buildah] (default "docker")

Global Flags:
      --verbose   Enable verbose logging

```

Reply:
Please re-test and let me know, as I can build successfully when i did a fresh fork to check.